VideoGame_Beginning

```diff
@@ -0,0 +1,17 @@
+Resaons for Argument
```

Comment:
Gaming trends and how predicted in the future. AI and critic scores

Reply:
Looks good!